engine/src/main/docker/run.sh

```diff
@@ -32,4 +32,4 @@ if [[ -n "${KRB5_USER}" ]]; then
     grep -q -F "$CRON_EXPR" /etc/crontab || echo "$CRON_EXPR" >> /etc/crontab
 fi
 
-java -Djava.security.egd=file:/dev/./urandom -jar /execution-engine.jar
\ No newline at end of file
+java --add-opens java.base/java.nio=ALL-UNNAMED -Djava.security.egd=file:/dev/./urandom -jar /execution-engine.jar
```

Comment:
Wasn't this spark/databricks problem?

Reply:
It's Snowflake related. Databricks requires `java.base/java.net=ALL-UNNAMED`